pontis-ui/src/components/Banner.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import StyledTransition from "./common/StyledTransition";
+import { ExclamationIcon, XIcon } from "@heroicons/react/outline";
+
+const Banner = () => {
+  const [isOpen, setIsOpen] = useState(true);
+  return (
+    <div className="fixed inset-x-0 bottom-0 z-50 pb-2 sm:pb-5">
```

Comment:
Oskar said it might be better to have the banner displayed at the top of the page instead of the bottom because the bottom reminds people of cookies. The top is more intrusive, though. Thoughts?